ui/src/message_popup/author/question_editing_components/question_text_test.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import {expect} from 'chai';
+import sinon from 'sinon';
+
+import QuestionText from './question_text.jsx';
+
+import TextField from 'material-ui/TextField';
+
+import {testQuestion} from '../../test_fixtures.js';
+
+function testProps(props){
+  return ({
+    originalText: testQuestion.text,
+    questionText: testQuestion.text + ' THIS HAS BEEN EDITED!',
+    onQuestionTextChange: sinon.spy(),
+    ...props
+  });
+}
+
+describe('<EditingComponent.QuestionText />', ()=>{
+  it('renders the original question text', ()=>{
+    const props = testProps();
+    const wrapper = shallow(<QuestionText {...props} />);
+    expect(wrapper.find(".originalQuestionText").text()).to.equal(props.originalText);
+    expect(wrapper.find(TextField)).to.have.length(1);
+    expect(wrapper.find(TextField).props().value).to.equal(props.questionText);
+  });
+  it('does not render original question text if not provided', ()=>{
+    const props = testProps({originalText: undefined});
+    const wrapper = shallow(<QuestionText {...props} />);
+    expect(wrapper.find(".originalQuestionText")).to.have.length(0);
+  });
+  it('returns the changed text', ()=>{
+    const props = testProps();
+    const wrapper = shallow(<QuestionText {...props} />);
+    wrapper.find(TextField).simulate('change', { target: { value: testQuestion.text } });
+    expect(props.onQuestionTextChange.callCount).to.equal(1);
+    expect(props.onQuestionTextChange.firstCall.args).to.deep.equal([{ target: { value: testQuestion.text } }]);
```

Comment:
I did this now.  I also ran into some problems with React invariant violations around `onSetChildren` and `onSetParent` when hot-patching tests with mocha-watch.  I suspected this was from calling `Routes.navigate` which I was surprised ever works.  I ended up mocking those out and also removing the mounting test in `questions_page_test` since these both would lead to the invariant.

This works around, but it'll be good to spend some time and get tests working that can do normal mounting, either with Enzyme or without.